models/patchArticles.model.js

```diff
@@ -0,0 +1,21 @@
+const db = require('../db/connection')
+const getArticleByID = require('../models/getArticleByID.model')
+
+function updateArticles (article_id, inc_votes) {
+    return getArticleByID(article_id).then((article) => {
+        if (!article){
+            return null
+        }
+
+    article.votes += inc_votes
+    const queryString = 'UPDATE articles SET votes = $1 WHERE article_id = $2 RETURNING *'
```

Comment:
We could have achieved this a little easier by using the 'SET votes = votes + $1' syntax. No need for a separate database query.